libs/template/presets/storefront/experience/pages/my-account/carts-page.ts

```diff
@@ -0,0 +1,68 @@
+import { ExperienceComponent } from '@spryker-oryx/experience';
+import { RouteType } from '@spryker-oryx/router';
+import { Size } from '@spryker-oryx/utilities';
+
+export const cartsPage: ExperienceComponent = {
+  id: 'carts-page',
+  type: 'Page',
+  meta: {
+    title: 'Carts',
+    route: '/my-account/carts',
```

Comment:
we need to use a `routeType` here, please align with @dunqan

Reply:
But it has been already added